README.md

```diff
@@ -1,4 +1,174 @@
+# Tic Tac Toe App
 
 It is implementation of TicTacToe game with simple AI opponent.
 I followed instuctions from JetBrains Academy https://hyperskill.org/projects/48
 
+## How to Build
+
+You can build this app executable jar with gradle wrapper unix executable with this command. Make sure you have jdk 8 installed in your computer.
+
+```sh
+./gradlew jar
+```
+
+Or if you're on windows you can use this command, this will run the batch file
+
+```sh
+gradlew jar
+```
+
+You can find the executable jar file in `build/libs/TicTacToe-1.0-SNAPSHOT.jar`
+
+## How to Run
+
+After building the executable jar, you can run the jar with this command
+
+```sh
+java -jar build/libs/TicTacToe-1.0-SNAPSHOT.jar
+```
+
+## How to start
+
+First you need to enter 3 parameter to start the game,
+
+- 1st parameter is game menu action, you can choose between:
+  - `start` to start the game
+  - `exit` to quit the game
+- 2nd parameter is player 1 with X symbol
+- 3rd parameter is player 2 with O symbol
+
+You can choose the player 1 or player 2 between these
+
+- `user` for human player
+- `easy` for easy difficulty bot
+- `medium` for medium difficulty bot
+- `hard` for hard difficulty bot
+
+So, for example if you want to play against easy you can type these
+
+```java
+Input command: start user easy
+```
+
+To make bot play first, you can type user as 3rd parameter, like this.
+
+```java
+Input command: start medium user
+```
+
+You can also let bot play against each other, like this.
+
+```java
+Input command: start hard hard
+```
+
+## Playing the game
+
+The rule is simple:
+
+1. The game requires two players, X and O.
+2. The game board is a set 3x3 grid in which players will place their symbol to claim that segment.
+3. X typically players first, then players alternate turns.
+4. The goal is to claim three segments of the grid in a row, either horizontally, vertically, or diagonally.
+No additional sides can be added to the grid.
+5. The game is over either when one player achieves three segments in a row, or when the grid is filled without anyone achieving three segments in a row.
+
+In this game, after you select `user` (human player) to play the game, you can type in the coordinates to play the game. You'll be receiving this input command.
+
+```java
```

Comment:
it is not java code

Reply:
should I change it to `sh`, since the input is in console?

Reply:
sure.